cmat128/matrix.go

```diff
@@ -35,7 +37,7 @@ type Conjugate struct {
 // At returns the value of the element at row i and column j of the transposed
 // matrix, that is, row j and column i of the Matrix field.
 func (t Conjugate) At(i, j int) complex128 {
-	return t.Matrix.At(j, i) * complex(1, -1)
+	return cmplx.Conj(t.Matrix.At(j, i))
```

Comment:
In fact this is what I originally had there but then I noticed that cmplx.Conj exists without checking what it does internally. Done.